### ISSUE TITLE
Dashboard structured log filter truncates value at space

### ISSUE THREAD

**Issue by KalleOlaviNiemitalo** [dotnet/aspire#2846]
## To reproduce

1. Start the Aspire dashboard standalone.  On Windows:
   ```
   CD "C:\Program Files\dotnet\packs\Aspire.Dashboard.Sdk.win-x64\8.0.0-preview.4.24156.9\tools"
   .\Aspire.Dashboard.exe
   ```
3. Open the dashboard <http://localhost:18888> in a Web browser.
4. It redirects to the "Structured logs" page <http://localhost:18888/structuredlogs>.
5. Click the icon at the right side of the "Filters: No filters" text near the top right corner of the page.
6. The "Add filter" pane opens at the right side of the page.  It has fields "Message", "contains", and "Value", and buttons "Cancel" and "Apply filter".
7. Click the "Value" field and type "hello world".
8. Click the "Apply filter" button.

## Expected result

The page should show "Filters: Message contains hello world".

## Actual result

The page shows "Filters: Message contains hello".  There is no "world".

## Notes

If you then edit the filter, change the value from "hello" to "hello world", and click "Apply filter", then the dashboard keeps "world" too.

This issue occurs regardless of whether there is a program publishing telemetry to the dashboard.

<https://github.com/dotnet/aspire/issues/2762> looks a bit related but I'm not sure it's the same bug.

**Comment by tlmii** [comment]
@adamint Is this a dupe of #2762 that will be fixed by #2817?

**Comment by adamint** [comment]
Yes, it is. Closing
